logger.js

```diff
@@ -27,6 +27,7 @@ function pinoLogger (opts, stream) {
   function onResFinished (err) {
     this.removeListener('finish', onResFinished)
     this.removeListener('error', onResFinished)
+    this.req.removeListener('aborted', onReqAborted);
```

Comment:
Can we get away with not removing aborted listener?

If req is done, aborted won't fire and the whole object is heading for gc. I ask because removeListener isn't cheap